web/packages/test/scripts/deploy-ethereum.sh

```diff
@@ -6,7 +6,26 @@ source scripts/set-env.sh
 start_geth() {
     if [ "$eth_network" == "localhost" ]; then
         echo "Starting geth local node"
-        cp config/genesis.json $output_dir/genesis.json
+        local timestamp=""
+        if [[ "$(uname)" == "Darwin" && -z "${IN_NIX_SHELL:-}" ]]; then
+            if [ "$eth_fast_mode" == "true" ]; then
+                timestamp=$(gdate -d'+180second' +%s)
+            else
+                timestamp=$(gdate -d'+1000second' +%s)
+            fi
+        else
+            if [ "$eth_fast_mode" == "true" ]; then
+                timestamp=$(date -d'+180second' +%s)
+            else
+                timestamp=$(date -d'+1000second' +%s)
+            fi
+        fi
+        jq \
+            --argjson timestamp "$timestamp" \
+            '
+            .config.CancunTime = $timestamp
+            ' \
+            config/genesis.json >$output_dir/genesis.json
```

Comment:
The hack here is to test the switchover process. The point here is that we start with a `Shapella` setup and after 180 seconds switch to `Decun`, then we check that the beacon relayer works just as expected.